.github/workflows/build.yml

```diff
@@ -20,6 +19,17 @@ jobs:
       - name: Add repository to git safe directories
         run: git config --global --add safe.directory $GITHUB_WORKSPACE
 
+      - name: Check L1 geth version
+        run: ./ops/scripts/geth-version-checker.sh || (echo "geth version is wrong, update ci-builder"; exit 1)
+
+      - name: Git submodules
+        run: make submodules
+
+      - name: Set up Go
```

Comment:
Ci-builder already has go. Is it right?
I can't find to cache go.mod in this PR. Can you explain more ? 🙇🏻‍♂️

Reply:
yes,`actions/setup-go@v4` already had the caching layer, you can see, this step is trigger after everything is done

<img width="1195" alt="Screenshot 2024-06-13 at 09 41 07" src="https://github.com/tokamak-network/tokamak-thanos/assets/149703736/69628d83-870f-4a9b-bbec-c1b8fd938d0d">

Reply:
Same as you use @action/cache with the key is `key_name`